fluentui_others/build.gradle

```diff
@@ -36,13 +36,6 @@ android {
     }
     kotlinOptions {
```

Comment:
kotlinOptions added for V2 button. Remove this as well.

Reply:
done